gtsam/linear/Errors.h

```diff
@@ -31,29 +31,31 @@ namespace gtsam {
   class VectorValues;
 
   /** vector of errors */
-  class Errors : public FastList<Vector> {
+  class GTSAM_EXPORT Errors : public FastList<Vector> {
```

Comment:
According to `Using GTSAM_EXPORT.md`, I think I need it, as Errors has methods in .cpp file.

Reply:
I'll take your word for it, but just wanted to point out just in case, in `Using-GTSAM-EXPORT.md`, "Note that this implictly requires the class does not derive from a "header-only" class." (second bullet point:

https://github.com/borglab/gtsam/blob/00dc32df0dbbe783a9ccdc06dcd616369703075d/Using-GTSAM-EXPORT.md?plain=1#L7-L9